content/providers/01-ai-sdk-providers/170-baseten.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: Baseten
+description: Learn how to use Baseten models with the AI SDK.
+---
+
+# Baseten Provider
+
+[Baseten](https://baseten.ai/) is an inference platform for serving frontier, enterprise-grade open source AI models[API](https://docs.baseten.co/overview).
+
+## Setup
+
+The Baseten provider is available via the `@ai-sdk/baseten` module. You can install it with
+
+<Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tab>
+    <Snippet text="pnpm add @ai-sdk/baseten" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="npm install @ai-sdk/baseten" dark />
+  </Tab>
+  <Tab>
+    <Snippet text="yarn add @ai-sdk/baseten" dark />
+  </Tab>
+</Tabs>
+
+## Provider Instance
+
+You can import the default provider instance `baseten` from `@ai-sdk/baseten`:
+
+```ts
+import { baseten } from '@ai-sdk/baseten';
+```
+
+If you need a customized setup, you can import `createBaseten` from `@ai-sdk/baseten`
+and create a provider instance with your settings:
+
+```ts
+import { createBaseten } from '@ai-sdk/baseten';
+
+const baseten = createBaseten({
+  apiKey: process.env.BASETEN_API_KEY ?? '',
+});
+```
+
+You can use the following optional settings to customize the Baseten provider instance:
+
+- **baseURL** _string_
+
+  Use a different URL prefix for API calls, e.g. to use proxy servers.
+  The default prefix is `https://inference.baseten.co/v1`.
+
+- **apiKey** _string_
+
+  API key that is being sent using the `Authorization` header. It defaults to
+  the `BASETEN_API_KEY` environment variable. It is recommended you set the environment variable using `export` so you do not need to include the field everytime.
```

Comment:
maybe link to https://app.baseten.co/settings/api_keys from here or further up so folks have a direct link from where to get an API key?